src/contracts/core/DelegationManager.sol

```diff
@@ -115,30 +114,18 @@ contract DelegationManager is
      */
     function registerAsOperator(
         OperatorDetails calldata registeringOperatorDetails,
-        uint32 allocationDelay,
```

Comment:
we should still call into the AVSD imo

Reply:
not a fan but will delegate thoughts to @shrimalmadhur who will probably say yes

Reply:
I would like for it to be a single call when someone becomes operator - they can set it then only.
what's the reason for not having it here vs calling avs directory separately?